<ui version="4.0" >
 <class>UserFunctionDialog</class>
 <widget class="QDialog" name="UserFunctionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>User Function</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Function</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Expression</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QListWidget" name="lstCategory" />
     </item>
     <item row="1" column="2" >
      <widget class="QListWidget" name="lstFunction" />
     </item>
     <item row="1" column="3" >
      <widget class="QTextEdit" name="teExpression" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="undoRedoEnabled" >
        <bool>false</bool>
       </property>
       <property name="html" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="acceptRichText" >
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Parameters</string>
       </property>
       <property name="buddy" >
        <cstring>leParams</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leParams" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QTextEdit" name="teUserFunction" >
       <property name="tabChangesFocus" >
        <bool>true</bool>
       </property>
       <property name="acceptRichText" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QPushButton" name="btnAdd" >
         <property name="text" >
          <string>Add</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave" >
         <property name="text" >
          <string>Save</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemove" >
         <property name="text" >
          <string>Remove</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUse" >
         <property name="text" >
          <string>Use</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lstCategory</tabstop>
  <tabstop>lstFunction</tabstop>
  <tabstop>teExpression</tabstop>
  <tabstop>leParams</tabstop>
  <tabstop>teUserFunction</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnUse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
